packages/utils/src/indexBrowser.ts

```diff
@@ -0,0 +1,144 @@
+import allowAdditionalItems from './allowAdditionalItems';
+import asNumber from './asNumber';
+import canExpand from './canExpand';
+import createErrorHandler from './createErrorHandler';
+import createSchemaUtils from './createSchemaUtils';
+import dataURItoBlob from './dataURItoBlob';
+import dateRangeOptions from './dateRangeOptions';
+import deepEquals from './deepEquals';
+import englishStringTranslator from './englishStringTranslator';
+import enumOptionsDeselectValue from './enumOptionsDeselectValue';
+import enumOptionsIndexForValue from './enumOptionsIndexForValue';
+import enumOptionsIsSelected from './enumOptionsIsSelected';
+import enumOptionsSelectValue from './enumOptionsSelectValue';
+import enumOptionsValueForIndex from './enumOptionsValueForIndex';
+import ErrorSchemaBuilder from './ErrorSchemaBuilder';
+import findSchemaDefinition from './findSchemaDefinition';
+import getDateElementProps, { type DateElementFormat } from './getDateElementProps';
+import getDiscriminatorFieldFromSchema from './getDiscriminatorFieldFromSchema';
+import getInputProps from './getInputProps';
+import getSchemaType from './getSchemaType';
+import getSubmitButtonOptions from './getSubmitButtonOptions';
+import getTemplate from './getTemplate';
+import { getTestIdsBrowser as getTestIds } from './getTestIds';
```

Comment:
Why doesn't `getTestIds` work in the browser?

Reply:
Browsers don't have `process` defined and caused a TypeError